nvflare/app_opt/xgboost/histogram_based_v2/adaptors/grpc_client_adaptor.py

```diff
@@ -103,8 +105,10 @@ def __init__(
         self._run_dir = None
         self._process = None
         self._starter = None
+        self.engine = None
```

Comment:
we already have self.engine in the base class?

Reply:
You are right. The statement is removed.